lib/driver.js

```diff
@@ -259,10 +259,19 @@ class XCUITestDriver extends BaseDriver {
     log.info(`${this.opts.bundleId} is in foreground`);
   }
 
-  async startWdaSession (app, bundleId) {
+  async startWdaSession (app, bundleId, processArguments) {
+    let appArguments = [];
```

Comment:
These can be simplified to:

```js
let args = !!processArguments ? processArguments.args : [];
let env = !!processArguments ? processArguments.env : {};
```

We cannot (or, rather, should not) use `arguments` as a member of an object, since it is a keyword used by JavaScript. It probably won't fail, but it is confusing when reading it.